Example/Tests/UIColorPelotonTests.swift

```diff
@@ -59,6 +59,31 @@ final class UIColorPelotonSpec: QuickSpec {
       expect(view) == snapshot()
     }
 
+    it("pelotonHRZ1Blue looks right") {
+      view.backgroundColor = .pelotonHRZ1Blue
```

Comment:
I think we should name this either independent of the zone number or independent of the color. Meaning something like `.heartRateZone1` or `heartRateBlue`